recipe/meta.yaml

```diff
@@ -5,13 +5,11 @@ package:
   version: {{ version }}
 
 source:
-  url: https://github.com/libgeos/libgeos/archive/{{ version }}.tar.gz
-  sha256: e9e20e83572645ac2af0af523b40a404627ce74b3ec99727754391cdf5b23645
-  patches:
-    - xmltester.cpp.patch  # [win]
+  url: http://download.osgeo.org/geos/geos-{{ version }}.tar.bz2
```

Comment:
Would there be any reason why doing this would change the API? Are the tarballs identical apart from the `-dev` bit?

Reply:
yes, that's the main difference. The only other difference is paths in a doxygen file.